RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -1612,72 +1624,71 @@ PixelQuintupletsBuffer<alpaka_common::DevHost>* Event::getPixelQuintuplets(bool
   return pixelQuintupletsInCPU;
 }
 
-TrackCandidatesBuffer<alpaka_common::DevHost>* Event::getTrackCandidates(bool sync) {
-  if (trackCandidatesInCPU == nullptr) {
-    // Get nTrackCanHost parameter to initialize host based trackCandidatesInCPU
+::lst::TrackCandidatesHostCollection* Event::getTrackCandidates(bool sync) {
+  if (trackCandidatesHC_ == nullptr) {
```

Comment:
Yeah, I think this PR will solve the issue I was having.

Are we planning to switch everything to SoA+PortableCollection or only track candidates? If not everything will be switched we'll still have the same issue with other classes, but it might be fine to have a bit of "ugly" code that will only be used in standalone.

Reply:
I wanted to complete one case before migrating the rest.